.prettierrc.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
```

Comment:
Are these what we were already using or is this something completely new?

Reply:
It's what we were using (but it includes json formatting abilities). Changes to this will be picked up in eslint.